.vscode/launch.json

```diff
@@ -2,21 +2,17 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "type": "node2",
+      "type": "node",
```

Comment:
We originally switched to `node2` because it handled sourcemaps better, but it has a bug (presumably) that prevents the process from exiting when the last test finishes. We could live with it, but combining the `outFiles` property with `node` gives the desired behaviour for everything.